package lab2.task2;

public class BubbleSort {
    /**
     * Сортирует массив методом пузырька по возрастанию.
     *
     * @param arr массив целых чисел для сортировки
     */
    public static void sort(int[] arr) {
        int n = arr.length;

        for (int i = 0; i < n - 1; i++) {
            for (int j = 0; j < n - 1 - i; j++) {
                if (arr[j] > arr[j + 1]) {
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }
    }
}

